x-pack/plugins/apm/public/components/app/Main/routeConfig.tsx

```diff
@@ -6,10 +6,9 @@
 
 import { i18n } from '@kbn/i18n';
 import React from 'react';
-import { Redirect } from 'react-router-dom';
-
+import { Redirect, RouteComponentProps } from 'react-router-dom';
```

Comment:
Yay for RouteComponentProps, I didn't even realize it was also a generic<> :tada: